indexer/jobs/export_token_balances_job.py

```diff
@@ -66,22 +66,23 @@ def _collect_batch(self, parameters):
     def _process(self, **kwargs):
         if TokenBalance.type() in self._data_buff:
             self._data_buff[TokenBalance.type()].sort(key=lambda x: (x.block_number, x.address))
-
-            self._data_buff[CurrentTokenBalance.type()] = distinct_collections_by_group(
-                [
-                    CurrentTokenBalance(
-                        address=token_balance.address,
-                        token_id=token_balance.token_id,
-                        token_type=token_balance.token_type,
-                        token_address=token_balance.token_address,
-                        balance=token_balance.balance,
-                        block_number=token_balance.block_number,
-                        block_timestamp=token_balance.block_timestamp,
-                    )
-                    for token_balance in self._data_buff[TokenBalance.type()]
-                ],
-                group_by=["token_address", "address", "token_id"],
-                max_key="block_number",
+            self._update_domains(
```

Comment:
Does this work?

Reply:
```
def _update_domains(self, domains: List[Domain]):

        key = domains[0].type()
        self._data_buff[key] = domains
```
This is a simple code, just to prevent users from using self._data_buff directly